jwst/datamodels/schemas/pathloss.schema.yaml

```diff
@@ -1,6 +1,7 @@
 title: Pathloss correction data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: keyword_pexptype.schema.yaml
```

Comment:
PATHLOSS ref files use only EXP_TYPE, in an or-like fashion, as a CRDS selector, so this looks good. Are any other keywords from the ref file used in the pathloss step code?